### PR TITLE
Added the date display in my-badges.hbs

### LINKED ISSUE #1889: Storing the date of badge generation in database an showing on mybadges page
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Expected behaviour:**

The date at which badge was generated should also be stored and displayed on my-badges page.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by rajvaibhavdubey**
Fixes #1889 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Date of badge creation is now being displayed in the my-badges page. 
- Wrote helpers for extraction of Date and Time from the date object

### Screenshots for the change

Screenshot from 2019-03-21 23-19-21

### CODE REVIEW

**Comment by mariobehling** [comment]
Ultimately there should be an option to choose the event date format. Let's move there step by step. PLease resolve conflicts.

**Comment by rajvaibhavdubey** [comment]
@sk9331657 @mariobehling Resolved and also added the helpers for date and time splitting.

**Comment by sk9331657** [comment]
> @sk9331657 @mariobehling Resolved and also added the helpers for date and time splitting.

Add New Screenshot .

File: api/models/badges.py [hunk 1]
```diff
@@ -15,8 +15,8 @@ class Badges(db.Model):
     download_link = db.Column(db.String)
     image_link = db.Column(db.String)
     logo_image_link = db.Column(db.String)
-    created_at = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
-    deleted_at = db.Column(db.DateTime, nullable=True)
+    created_at = db.Column(db.String(100), nullable=False, default=datetime.utcnow())
```
Comment (yashLadha) [inline api/models/badges.py]: Should be `Date` only

File: api/schemas/badges.py [hunk 2]
```diff
@@ -55,7 +55,7 @@ class Meta:
     badge_id = fields.Str(required=True)
     text_color = fields.Str(required=True)
     badge_size = fields.Str(required=True)
-    created_at = fields.Date(required=True)
+    created_at = fields.Str(required=True)
```
Comment (yashLadha) [inline api/schemas/badges.py]: This is wrong

**Pull request closed without merging**